twikit/client/client.py

```
@@ -1238,6 +1238,11 @@ async def create_tweet(
             reply_to, attachment_url, community_id, share_with_followers,
             richtext_options, edit_tweet_id, limit_mode
         )
+        if 'errors' in response.keys():
```

Ruff (0.7.0)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)